priv/templates/phx.gen.auth/login_live.ex

```diff
@@ -0,0 +1,47 @@
+defmodule <%= inspect context.web_module %>.<%= inspect Module.concat(schema.web_namespace, schema.alias) %>LoginLive do
+  use <%= inspect context.web_module %>, :live_view
+
+  def render(assigns) do
+    ~H"""
+    <h1>Log in</h1>
+
+    <.form
+      id="login_form"
+      :let={f}
+      for={:<%= schema.singular %>}
+      phx-change="validate"
```

Comment:
We should remove validate. :)

Reply:
note to self, removing validate is pending on me with a new LV feature that maintains form state in the event of phx-change absence